[0.248s] Invoking command in '/home/ryo/ros2_ws/build/person_msg': CMAKE_PREFIX_PATH=/home/ryo/ros2_ws/install/mypkg:/opt/ros/humble /usr/bin/cmake /home/ryo/ros2_ws/src/person_msg -DCMAKE_INSTALL_PREFIX=/home/ryo/ros2_ws/install/person_msg
[0.263s] -- Found ament_cmake: 1.3.2 (/opt/ros/humble/share/ament_cmake/cmake)
[0.454s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.456s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.460s] -- Found rosidl_adapter: 3.1.3 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.494s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.771s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.771s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.771s]   person_msg --arguments-file
[0.771s]   /home/ryo/ros2_ws/build/person_msg/rosidl_adapter__arguments__person_msg.json
[0.771s]   --output-dir /home/ryo/ros2_ws/build/person_msg/rosidl_adapter/person_msg
[0.771s]   --output-file
[0.771s]   /home/ryo/ros2_ws/build/person_msg/rosidl_adapter/person_msg.idls) returned
[0.771s]   error code 1:
[0.771s] 
[0.771s]   Error processing 'unit8 age' of 'person_msg/Person': ''unit8' is an invalid
[0.772s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[0.772s] 
[0.772s]   Traceback (most recent call last):
[0.772s] 
[0.772s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.772s]       return _run_code(code, main_globals, None,
[0.772s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.772s]       exec(code, run_globals)
[0.772s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.772s]       sys.exit(main())
[0.772s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.772s]       abs_idl_file = convert_to_idl(
[0.772s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.772s]       return convert_msg_to_idl(
[0.772s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.772s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.773s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.773s]       Type(type_string, context_package_name=pkg_name),
[0.773s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.773s]       super(Type, self).__init__(
[0.773s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.773s]       raise InvalidResourceName(
[0.773s] 
[0.773s]   rosidl_adapter.parser.InvalidResourceName: 'unit8' is an invalid message
[0.773s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.773s] 
[0.773s] Call Stack (most recent call first):
[0.773s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.773s]   CMakeLists.txt:11 (rosidl_generate_interfaces)
[0.773s] 
[0.773s] [0m
[0.773s] -- Configuring incomplete, errors occurred!
[0.773s] See also "/home/ryo/ros2_ws/build/person_msg/CMakeFiles/CMakeOutput.log".
